### PR TITLE
Produce an Alpine Linux version of the ASP.NET Core shared framework

### PR DESCRIPTION
**Pull request by natemcmaster**
Part of #833. Support for Alpine is still going to be experimental. This will unblock being able to produce a shared runtime installer for alpine so we can start figuring out if there are other dependencies we need to upgrade. For sure we will need to update libuv, but it may be possible to rely on the libuv baked into Alpine Linux for now.

Changes in this PR:
 - Update to SQLitePCLRaw 1.1.10-pre20180223200113 (cc @Eilon for OSS approval)
 - Add alpine.3.6-x64 as a new RID
 - Add new external dependencies

Incidentally resolves https://github.com/aspnet/Universe/issues/843 (FYI @bricelam )
See also: https://github.com/aspnet/Microsoft.Data.Sqlite/pull/488

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -179,9 +184,9 @@
     <RemotionLinqPackageVersion>2.2.0</RemotionLinqPackageVersion>
     <SerilogExtensionsLoggingPackageVersion>1.4.0</SerilogExtensionsLoggingPackageVersion>
     <SerilogSinksFilePackageVersion>3.2.0</SerilogSinksFilePackageVersion>
-    <SQLitePCLRawBundleGreenPackageVersion>1.1.8</SQLitePCLRawBundleGreenPackageVersion>
-    <SQLitePCLRawBundleSqlcipherPackageVersion>1.1.8</SQLitePCLRawBundleSqlcipherPackageVersion>
-    <SQLitePCLRawCorePackageVersion>1.1.8</SQLitePCLRawCorePackageVersion>
+    <SQLitePCLRawBundleGreenPackageVersion>1.1.10-pre20180223200113</SQLitePCLRawBundleGreenPackageVersion>
```
Comment (JunTaoLuo) [inline build/dependencies.props]: Where is this version coming from? I don't see it on nuget.

Comment (natemcmaster) [reply build/dependencies.props]: https://www.nuget.org/packages/SQLitePCLRaw.bundle_green/1.1.10-pre20180223200113 ?

Comment (JunTaoLuo) [reply build/dependencies.props]: huh, not sure why i didn't see it before

Comment (JunTaoLuo) [reply build/dependencies.props]: Still seems like a very strange version number. I'm assuming there will be a 1.1.10 stable at some point?

Comment (natemcmaster) [reply build/dependencies.props]: Yes, Eric said he was planning to have 1.1.10 ready sometime in the next few weeks.

**Comment by natemcmaster** [comment]
We can use https://github.com/aspnet/Universe/issues/933 to track follow-up to move to SQLitePCLRaw 1.1.10 when it becomes available.

**Pull request merged**